diagnostic.md

```diff
@@ -3,53 +3,50 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+div id "js-pjax-loader-bar"
+class "pjax-loader-bar"
+div id "start-of-content"
+div id "ajax-error-message"
```

Comment:
These should be written as `div#js-pjax-loader-bar`, `.pjax-loader-bar`, `div#start-of-content`, and `div#ajax-error-message`.